internal/connector/internal/handlers/processor_secrets.go

```diff
@@ -0,0 +1,30 @@
+package handlers
+
+import (
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/connector/internal/api/dbapi"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/connector/internal/services/vault"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
+)
+
+const OwningProcessorResourcePrefix = "/v1/processor/"
+
+func stripProcessorSecretReferences(resource *dbapi.Processor) *errors.ServiceError {
```

Comment:
There are (AFAIK) no "secret" parameters in Camel-K DSL that we'd need to strip out and store in the Vault.

I only store the kafka credentials.

Reply:
At this stage I won't expect any processor to include secrets in the dsl but to use the service account that is provided so I would even remove this method for now.